### PR TITLE
feat: properly highlight subscription cell

### PR DESCRIPTION
**Pull request by legendecas**
@RocketChat/iOS

### CODE REVIEW

**Comment by legendecas** [comment]
@rafaelks looks like there is something wrong with check phase `licence/cla`.

File: Scripts/swiftlint.sh [hunk 1]
```diff
@@ -5,11 +5,11 @@ function lint() {
 	swiftlint
 }
 
-if which swiftlint >/dev/null; then
+if hash swiftlint 2>/dev/null; then
   lint
 else
     # lets try to install swift lint using homebrew
-    if which brew >/dev/null; then
+    if hash brew 2>/dev/null; then
```
Comment (rafaelks) [inline Scripts/swiftlint.sh]: Can you remove this changes? I think it's not necessary anymore

File: Rocket.Chat/Views/Cells/Subscription/SubscriptionCell.swift [hunk 2]
```diff
@@ -16,6 +16,10 @@ final class SubscriptionCell: UITableViewCell {
     internal let labelReadTextColor = UIColor(rgb: 0x9AB1BF, alphaVal: 1)
     internal let labelUnreadTextColor = UIColor(rgb: 0xFFFFFF, alphaVal: 1)
 
+    internal let defaultBackgroundColor = UIColor.clear
+    internal let selectedBackgroundColor = UIColor(rgb: 0x0, alphaVal: 0.18)
+    internal let highlightedBackgroundColor = UIColor(rgb: 0x0, alphaVal: 0.27)
```
Comment (rafaelks) [inline Rocket.Chat/Views/Cells/Subscription/SubscriptionCell.swift]: What are these colors? Maybe we could add it to the UIColorExtensions?

Comment (legendecas) [inline Rocket.Chat/Views/Cells/Subscription/SubscriptionCell.swift]: @rafaelks I think these colors are highly related to this component, so I put these here.

Comment (rafaelks) [inline Rocket.Chat/Views/Cells/Subscription/SubscriptionCell.swift]: Sounds good.

**Pull request merged**